docs/elements.md

```diff
@@ -990,15 +990,21 @@ def generate(data):
+| `remove-leading-trailing` | boolean             | false    | Whether or not to remove leading and trailing blank spaces from the input string. Default is `true` if `multiline` is enabled.                                            |
 | `remove-spaces`           | boolean             | false    | Whether or not to remove blank spaces from the input string.                                                                                                              |
 | `allow-blank`             | boolean             | false    | Whether or not an empty input box is allowed. By default, empty input boxes will not be graded (invalid format).                                                          |
 | `ignore-case`             | boolean             | false    | Whether or not to enforce case sensitivity (e.g. "hello" != "HELLO").                                                                                                     |
 | `normalize-to-ascii`      | boolean             | false    | Whether non-English characters (accents, non-latin alphabets, fancy quotes) should be normalized to equivalent English characters before submitting the file for grading. |
 | `placeholder`             | text                | None     | Hint displayed inside the input box describing the expected type of input.                                                                                                |
-| `size`                    | integer             | 35       | Size of the input box.                                                                                                                                                    |
+| `size`                    | integer             | 35       | Width of the input box.                                                                                                                                                   |
 | `show-help-text`          | boolean             | true     | Show the question mark at the end of the input displaying required input parameters.                                                                                      |
+| `multiline`               | boolean             | false    | Whether or not to allow the input to include line breaks.                                                                                                                 |
+| `escape-unicode`          | boolean             | true     | Whether or not to escape unprintable unicode characters. Default is `false` if `multiline` is enabled.                                                                    |
+
+#### Using multiline inputs
+
+Note that, in multiline inputs, it can be hard to distinguish between inputs with or without a terminating line break (i.e., an additional "Enter" at the end of the input). Because of that, you are strongly encouraged to use `remove-leading-trailing="true"` when using multiline inputs.
```

Comment:
```suggestion
Note that, in multiline inputs, it can be hard to distinguish between inputs with or without a terminating line break (i.e., an additional "Enter" at the end of the input). Because of that, you are strongly encouraged to leave the default setting of `remove-leading-trailing="true"` unchanged when using multiline inputs.
```